WORKSPACE

```diff
@@ -18,24 +18,14 @@ register_toolchains(
     "//third_party/toolchains/bazel_0.26.0_rbe_windows:cc-toolchain-x64_windows",
 )
 
-load("@bazel_toolchains//rules/exec_properties:exec_properties.bzl", "create_exec_properties_dict", "custom_exec_properties", "merge_dicts")
+load("@bazel_toolchains//rules/exec_properties:exec_properties.bzl", "create_rbe_exec_properties_dict", "create_exec_properties_dict", "custom_exec_properties", "merge_dicts")
```

Comment:
You no longer need to load any of "create_rbe_exec_properties_dict", "create_exec_properties_dict", "custom_exec_properties", "merge_dicts"?